Documentation/build-configure.md

```diff
@@ -0,0 +1,137 @@
+# Build configuration
+
+## Overview
+
+This file describes all the parameters of the configure script and their possible uses.
+For a quick help of available parameters, run `./configure --help`.
+
+## Parameters for building stage1 flavors
+
+#### `--with-stage1-flavors`
+
+This parameter takes a comma-separated list of all the flavors that the build system should assemble.
+Depending on a default stage1 image setup, this list is by default either empty or set to `coreos,kvm` for, respectively, detailed setup and flavor setup.
+Note that specifying this parameter does not necessarily means that rkt will use them in the end.
+Available flavors are:
+
+- `coreos` - it takes systemd and bash from CoreOS PXE image; uses systemd-nspawn
+- `kvm` - it takes systemd, bash and other binaries from CoreOS PXE image; uses lkvm
+- `src` - it builds systemd, takes bash from host at build time; uses built systemd-nspawn
+- `host` - it takes systemd and bash from host at runtime; uses systemd-nspawn from host
+
+The `host` flavor is probably best suited flavor for distributions that have strict rules about software sources.
+
+#### `--with-stage1-flavors-version-override`
+
+This parameter takes a version number to become a version of all the built flavors of stage1 image.
```

Comment:
"This parameter takes a version number to become the version of all the built stage1 flavors."